src/components/pages/Memos/Memos.css

```diff
@@ -28,6 +33,10 @@
   padding: 1em 0.5em;
 }
 
+.ant-radio-checked .ant-radio-inner::after {
```

Comment:
what does '::' do? I've never seen a double colon before.

Reply:
It means pseudo element selector. Which means the element to the right doesn't exist in the normal DOM, but can be selected and modified.